tests/test_s3urlcache.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import logging
+import os
+import sys
+import unittest
+
+from dss.storage.validator import scrub_index_data, SizeLimitError
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from dss import Config, BucketConfig
+from dss.config import IndexSuffix, ESDocType, Replica
+from tests.infra import start_verbose_logging
+from hashlib import sha1
+from dss.storage.validator import S3UrlCache
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+start_verbose_logging()
+
+def setUpModule():
+    IndexSuffix.name = __name__.rsplit('.', 1)[-1]
+
+
+def tearDownModule():
+    IndexSuffix.reset()
+
+
+class TestS3UrlCache(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.replica = Replica.aws
```

Comment:
This is only used in this method here and doesn't need to be a class attribute.

Reply:
fixed